src/session/cookie-store/index.ts

```diff
@@ -71,15 +83,32 @@ export default class CookieSessionStore implements ISessionStore {
     }
 
     const encryptedSession = await Iron.seal(persistedSession, cookieSecret, Iron.defaults);
-    setCookie(req, res, {
-      name: cookieName,
-      value: encryptedSession,
+
+    const cookies = [];
+    let start = 0;
+    do {
+      cookies.push({
+        name: `${cookieName}.${cookies.length}`,
+        value: encryptedSession.slice(start, start + 4000),
+        path: cookiePath,
+        maxAge: cookieLifetime,
+        domain: cookieDomain,
+        sameSite: cookieSameSite
+      });
+      start += 4000;
```

Comment:
I still seem to get too large cookies with 4000.

Reply:
Maximum cookie size per [rfc](https://tools.ietf.org/html/rfc6265) is 4096 bytes including cookie name, value and attributes. So yes, with long name, or additional properties it can be exceeded. @johansundwall do you have a recommendation what we should set here?

Reply:
I would wait till the maintainers give the green light on the possibility of merging this PR.

Maybe something like this https://stackoverflow.com/a/57071072/1593363 would be more reliable when splitting into chunks.

Reply:
You are right, this value can be tuned any time. Since the rebuild code does not depend on the slice size, it could be changed even on a running application.

Your second remark is about the Buffer class instead of strings?